resources/views/livewire/order/order-positions.blade.php

```diff
@@ -80,6 +80,15 @@ class="pb-4"
                                         :label="__('Vat rate')"
                                         wire:model.live="orderPosition.vat_rate_id"
                                     />
+                                    <template x-for="discount in $wire.orderPosition.discounts">
+                                        <x-input
+                                            prefix="%"
```

Comment:
why does this have the '%' prefix? It can either be a percentage discount or a flat discount.

Reply:
everted this, i will create a new PR for the "bulk add"

Reply:
If this would be reverted, why is it still showing as a change?